metricbeat/helper/http.go

```diff
@@ -0,0 +1,85 @@
+package helper
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/elastic/beats/metricbeat/mb"
+)
+
+type Http struct {
```

Comment:
I don't see anything wrong with `http *helper.HTTP`. It's not much different from `http *helper.Http`. Or am I missing something?